lib/model/katex.dart

```diff
@@ -136,6 +154,110 @@ class _KatexParser {
   KatexNode _parseSpan(dom.Element element) {
     // TODO maybe check if the sequence of ancestors matter for spans.
 
+    final spanClasses = List<String>.unmodifiable(element.className.split(' '));
+
+    if (element case dom.Element(localName: 'span', :final className)
+        when className.startsWith('vlist')) {
+      switch (element) {
+        case dom.Element(
+          localName: 'span',
+          className: 'vlist-t',
+          attributes: final attributesVlistT,
+          nodes: [
+            dom.Element(
+              localName: 'span',
+              className: 'vlist-r',
+              attributes: final attributesVlistR,
+              nodes: [
+                dom.Element(
+                  localName: 'span',
+                  className: 'vlist',
+                  nodes: [
+                    dom.Element(
+                      localName: 'span',
+                      className: '',
+                      nodes: [
+                        dom.Element(localName: 'span', className: 'pstrut')
+                          && final pstrutSpan,
+                        ...,
+                      ]) && final innerSpan,
+                  ]),
+              ]),
+          ])
+        when !attributesVlistT.containsKey('style') &&
+            !attributesVlistR.containsKey('style'):
+          // TODO vlist element should only have `height` style, which we ignore.
+
+          var styles = _parseSpanInlineStyles(innerSpan)!;
+          final topEm = styles.topEm ?? 0;
+
+          final pstrutStyles = _parseSpanInlineStyles(pstrutSpan)!;
+          final pstrutHeight = pstrutStyles.heightEm ?? 0;
+
+          // TODO handle negative right-margin inline style on row nodes.
```

Comment:
Do we consider this complete after the final commit ("content: Support negative right-margin on KaTeX spans")? I'm not too sure if something else is left to be done here.

This seems to contradict with the earlier assumption that vlist elements contain only the `height` style.